langdetect/detector_factory.py

```diff
@@ -32,8 +35,9 @@ def __init__(self):
         self.word_lang_prob_map = {}
         self.langlist = []
 
-    def load_profile(self, profile_directory):
-        list_files = os.listdir(profile_directory)
+    def load_profile(self, profile_directory, supported_languages=[]):
```

Comment:
It might also be nice to indicate somewhere (here?) what the contents of `supported_languages` should look like. i.e. what should each element of the list look like

Reply:
Oh, great suggestion! Will update this :)